src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -199,10 +200,16 @@
 	private CompilationUnit sourceCompilationUnit;
 
 	/**
+	 * Imports computed
+	 */
```

Comment:
longer name and documentation to explain the intention and usage of this new field?

should it be reset for each new type?